arch/x86_64/mcount-insn.c

```diff
@@ -193,6 +193,209 @@ static int handle_pic_lea(cs_insn *insn, uint8_t insns[],
 	return -1;
 }
 
+/*
+ *  handle PIC code for MOV instruction.
+ *  this function manipulate the instruction.
+ *
+ *  for examples :
+ *
+ *  case LOAD:
+ *	mov rcx, qword ptr [rip + 0x8f3f85]
+ *  to manipulate like below:
+ *	push rax
+ *	mov rax, PC_BASE + 0x8f3f85
+ *	mov rcx, [rax]
+ *	pop rax
```

Comment:
Can't we do just like below?

```
mov rcx, PC_BASE + 0x8f3f85
mov rcx, [rcx]
```

At least my simple program has following ...
```
    122b:	48 8b 40 08          	mov    rax,QWORD PTR [rax+0x8]
```

Reply:
@namhyung because registers except RAX only can take 4byte for displacement.

Reply:
Hey @ParkHanbum I think with prefix REX.w we can have an eight-byte disp :
0:  48 b9 88 77 66 55 44    movabs rcx,0x1122334455667788
7:  33 22 11

Reply:
@AnsBal you right, maybe it is better choice that using calculated address(imm64) than register base instruction. @namhyung how do you think?

Reply:
I even was not aware of the RAX limitation. :)  I'm fine with anything works!